gourmet/gtk_extras/TextBufferMarkup.py

```diff
@@ -78,26 +80,35 @@ def __init__ (self):
         GObject.GObject.__init__(self)
 
     def set_text (self, txt):
+        if isinstance(txt, bytes):
+            # data loaded from the database are bytes, not str
+            txt = txt.decode()
         Gtk.TextBuffer.set_text(self,"")
         try:
-            self.parsed,self.txt,self.separator = Pango.parse_markup(txt,'\x00')
-        except:
-            print('Problem encountered escaping text: "%s"'%txt)
+            self.parsed, self.attrList, self.txt, self.separator = Pango.parse_markup(txt, -1, '\x00')
```

Comment:
`parse_markup` now requires an extra parameter, (here harcoded as `-1`) indicating where does the markup text ends in the string.

In our cases, the whole string is markup and should be converted.

What's returned are:
- `self.parsed`: a success boolean;
- `self.attrList`: the list of attributes (bold, italic, ...), where each start and end;
- `self.txt`: the actual text iself, without markup;
- `self.separator`: the html separator ('\x00').